### ISSUE TITLE
slidable button for shows times like user User change role button and block ,unblock button

### ISSUE THREAD

**Issue by gunasekharravilla** [hoc081098/Movie-Ticket-Booking#65]
On the Manager  showtimes page, it's nice to have a button to  change the status of the show, slide button like on users pages, If the admin wants to stop the reservation for some time he can change the status of the showtime  and show the users message like currently, reservations are not accepting or if reservations stopped show message as Bookings closed
